apps/frontend/src/app/pages/company/applications-job/applications-job.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="container mx-auto min-h-screen px-3">
+  <div class="pb-4 pt-6 flex justify-between flex-wrap gap-4">
```

Comment:
Justify between works fine for look like the figma when we have 4 cards, but test with 2 or 3, it does not work good and for the user will be bad, maybe a justify-start should be an option